modules/azure-landing-zone/frontdoor_kv_access.tf

```diff
@@ -0,0 +1,10 @@
+resource "azurerm_key_vault_access_policy" "frontdoor_kv_access" {
+  key_vault_id = data.azurerm_key_vault.certificate_vault.id
+
+  object_id = "0a7e9367-4349-4b24-974a-bfa7b23a38fc"
```

Comment:
Instead of hardcoding this object id can we use something like below:
`data "azurerm_client_config" "current" {}`

and then 
`data.azurerm_client_config.current.object_id`

Reply:
it's not the current client it's the azure front door object id